message-processors/commodityv3-processor/src/main/java/io/edpn/backend/messageprocessor/commodityv3/application/service/SynchronizedReceiveCommodityMessageService.java

```diff
@@ -83,45 +71,34 @@ public synchronized void receive(CommodityMessage.V3 commodityMessage) {
         LOGGER.trace("ReceiveCommodityMessageService.receive -> took " + (System.nanoTime() - start) + " nanosecond");
```

Comment:
wrap this in a if(log.isTraceEnabled()) to not do unnecessary calculation and concatenation